# package-third-party-numpy-subpackage-random-C-extension-function-uniform
> [TABLE OF CONTENTS](https://nbviewer.jupyter.org/github/SeanOhAileasa/fubar-python/blob/main/fubar-python.ipynb#top)
***

In [1]:
# b_T_hird_P_arty-imports
from numpy.random import uniform as unf

In [2]:
# package numpy - subpackage random - C function uniform alias unf - copy - attribute __doc__ - access
print(unf.__doc__)


        uniform(low=0.0, high=1.0, size=None)

        Draw samples from a uniform distribution.

        Samples are uniformly distributed over the half-open interval
        ``[low, high)`` (includes low, but excludes high).  In other words,
        any value within the given interval is equally likely to be drawn
        by `uniform`.

        .. note::
            New code should use the ``uniform`` method of a ``default_rng()``
            instance instead; see `random-quick-start`.

        Parameters
        ----------
        low : float or array_like of floats, optional
            Lower boundary of the output interval.  All values generated will be
            greater than or equal to low.  The default value is 0.
        high : float or array_like of floats
            Upper boundary of the output interval.  All values generated will be
            less than high.  The default value is 1.0.
        size : int or tuple of ints, optional
            Output shape.  If the give

In [3]:
# package numpy - subpackage random - C function uniform alias unf - arguments
nLow,nHigh,nSize=-3.0,3.0,1000

In [4]:
# package numpy - subpackage random - C function uniform alias unf - call
unf(nLow,nHigh,nSize)

array([ 1.91772525, -2.18131091,  2.21226538,  2.81776931,  2.69198212,
       -2.20787041,  1.9057509 , -1.35677775, -2.42783505, -2.08187573,
        1.40958647,  2.6183904 ,  2.45539305, -0.09671168, -0.72643183,
       -1.23033064, -0.56711551, -1.30752265, -1.53660615,  2.78665036,
        2.84250949,  0.0217841 , -0.23903626,  1.03179646, -1.1327131 ,
       -2.89963067,  0.21302902, -1.85065497, -0.66340204,  1.93428983,
       -0.68963593, -2.04828458, -0.93280544,  1.1281877 , -0.64901257,
        2.65061046,  2.23186528, -2.45408832, -0.19429964,  0.27719501,
        1.82284049,  1.82590138,  1.99303249, -1.05890607,  1.73361466,
        1.605193  ,  0.70518657,  1.4318211 ,  2.5393888 , -0.46492457,
        2.26266568,  1.69453302, -2.6609796 , -1.31334878, -1.55247995,
       -2.29696493, -2.34372158,  0.72525842,  0.41613753,  2.48704422,
        0.30162995,  0.3860612 ,  2.68888408,  1.3770452 ,  0.39585757,
        1.84160731, -2.44859062, -0.20440252,  1.47254985,  2.54

- most of the values - ``1000`` - if plotting histogram - expect straight line at the top of the histogram [2.2]

***
# END